lib/src/managers/photo_manager.dart

```diff
@@ -57,6 +57,35 @@ class PhotoManager {
     return plugin.requestPermissionExtend(requestOption);
   }
 
+  /// Get the current permission state of the application
+  ///
+  /// This method returns a [Future] that resolves to the current [PermissionState] of the application.
+  ///
+  /// Possible values for [PermissionState] include:
+  /// - [PermissionState.notDetermined]: The user hasn't made a choice yet
+  /// - [PermissionState.restricted]: The application is restricted from accessing the photo library (iOS only)
+  /// - [PermissionState.denied]: The user has denied permission
+  /// - [PermissionState.authorized]: The user has granted full access permission
+  /// - [PermissionState.limited]: The user has granted limited access permission (iOS 14+ only)
+  ///
+  /// Example:
+  /// ```dart
+  /// PermissionState state = await PhotoManager.getPermissionState();
+  /// if (state == PermissionState.authorized) {
+  ///   print('The application has full access permission');
+  /// } else {
+  ///   print('The application does not have full access permission');
+  /// }
+  /// ```
+  ///
+  /// Note: On Android, this method may require an [Activity] context. If called in a background service,
+  /// make sure to call the [setIgnorePermissionCheck] method first.
+  static Future<PermissionState> getPermissionState({
+    PermissionRequestOption requestOption = const PermissionRequestOption(),
+  }) {
```

Comment:
Could we make `requestOption` as a required named parameter? In that way we let developers always have a sense of what they are trying to do and provide valid input.